pkgs/desktops/gnome/extensions/argos/default.nix

```diff
@@ -1,14 +1,19 @@
-{ fetchFromGitHub, lib, stdenv }:
+{ fetchFromGitHub, lib, stdenv, unstableGitUpdater }:
+
+# There are no tagged releases anymore so we're using unstable
+# but having this keep getting updated for every upstream commit
+# seems pointless.
+# nixpkgs-update: no auto update
 
 stdenv.mkDerivation rec {
   pname = "argos";
-  version = "unstable-2023-09-26";
+  version = "unstable-2024-04-03";
 
   src = fetchFromGitHub {
     owner = "p-e-w";
     repo = "argos";
-    rev = "adfaa31e8c08f7b59e9492891a7e6f753c29b35e";  # https://github.com/p-e-w/argos/pull/150
-    hash = "sha256-st8AeMRtkvM4M/Z70qopjw9Yx0t9l0DsUke4ClQtcBU=";
+    rev = "0449229e11bc2bb5c66e6f1d8503635cdf276bcf";  # https://github.com/p-e-w/argos/pull/150
```

Comment:
The linked PR got merged

Reply:
All the relevant PRs are merged but there's no tagged release, I've updated the link to the GNOME 46 PR instead of the 45 one but if the preference is to just drop the link please let me know.

Reply:
I believe that was done because the commit hash pointed to an unmerged PR.

Reply:
Removed the link.